niworkflows/common/report.py

```diff
@@ -23,6 +23,8 @@ class ReportCapableInputSpec(BaseInterfaceInputSpec):
         False, usedefault=True, desc="Set to true to enable report generation for node")
     out_report = File(
         'report.html', usedefault=True, desc='filename for the visual report')
+    mock_run = traits.Bool(False, usedefault=True,
```

Comment:
Could this be a private ```ReportCapableInterface``` member? I think it makes more sense that way.

- this is just a detail, this could perfectly be merged without this change.